spec/factories/authentication_tokens.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
```

Comment:
is this needed if the specs can use `AuthenticationToken.create_with_random_token`?

Reply:
Yeah makes sense, I'll remove 👍

Reply:
Actually @stevehook, I've just tried to do this and we need to be able specify what the unhashed version of the token is in some cases. This is so that we can use it explicitly in shared examples, like [this line here](https://github.com/DFE-Digital/register-trainee-teachers/blob/4999ba902bc332f169b003c21ae9f3514267b60d/spec/requests/api/v0.1/trainees_spec.rb#L12). I'd say it's worth keeping the factory around for now, let me know what you think.

Reply:
@AdColvin I see what you mean. The only other way around this would be to have the shared context set up its own provider and token. As the shared context always needs those things it makes some sense to me to encapsulate them.